### ISSUE TITLE
FDI on Intranet

### ISSUE THREAD

**Issue by C13L0** [phsc-edu/2018#268]
From: Margie Burnham
Sent: Thursday, May 31, 2018 10:30 AM
To: Webmaster
Cc: Sonia Thorn; Melissa Harts; Stanley Giannet
Subject: Re: FDI on Intranet
 
Cielo,

Thank you so much for the clarification. Please then post the June 2018 Faculty Development Institute Newsletter on the Intranet Announcements.  We publish a newsletter at the end of each month.

See below responses to the Webmaster questions:

Is the request for the intranet or main phsc.edu website? Intranet
Announcement title:  June 2018 Faculty Development Institute Newsletter
Announcement message: Comments or questions can be directed to Margie Bunham burnham@phsc.edu, 3227.
Attach any associated file attachment(s) in .pdf format PDF Attached to this email
Provide URL for page to modify. N/A
Additional detailed instructions N/A

Thank you so much, Cielo.

Margie

**Comment by C13L0** [comment]
[june2018b.pdf](https://github.com/phsc-edu/2018/files/2073895/june2018b.pdf)

**Comment by curtiskphsc** [comment]
@C13L0 I drafted a bit of copy for this. Please have it reviewed before posting. Also, they have at least one broken link in the pdf. The "Inside Higher Ed free subscription link" is not working. Looks like they truncated the actual hyperlink. Please link the pdf (after link correction) to the applicable text in the last line.
**June 2018 Faculty Development Institute Newsletter**
The June 2018 issue of the Faculty Development Institute Newsletter is now available! This month's edition features an announcement from Dean Thorn regarding a new course evaluation system that will be in place for the Fall 2018 semester, free NISOD webinar invites, and a free subscription to _Inside Higher Ed_ 
Read the June 2018 Faculty Development Institute Newsletter and for comments or questions contact Margie Bunham at burnham@phsc.edu or 3227.

**Comment by C13L0** [comment]
@curtiskphsc The link works fine for me...

------
From: Margie Burnham
Sent: Tuesday, June 5, 2018 05:40 PM
To: Webmaster
Cc: Sonia Thorn
Subject: Re: FDI on Intranet | 268 | Awaiting further info
 

Thank you so much.  I really like the verbiage that Marketing created.  
I cannot fix the link.  Apparently there is a problem with the hyperlink and our server allowing such.  I've attached the new file that I tried to fix.

Margie

[june2018b-revision.pdf](https://github.com/phsc-edu/2018/files/2076188/june2018b-revision.pdf)
